data/ocr/regex_brands.txt

```diff
@@ -48,11 +56,14 @@ Gerblé
 Grand Jury
 Gut & Gunstig
 Gutbio
+Häagen-Dazs||H[aä]agen[- ]Dazs
```

Comment:
@teolemon the regex part must be in lowercase

Reply:
fixed